tests/unit/test_handle_custom_resource_message.py

```diff
@@ -0,0 +1,64 @@
+from restrict_download_region import cfnresponse, restrict_region
+from pytest_mock import MockerFixture
+import pytest
+
+
+def test_handle_custom_resource_status_message__successful_execution__not_custom_resource(mocker: MockerFixture):
+    mock_cfn_send = mocker.patch.object(cfnresponse, "send", autospec=True)
+
+    event = {}
+    context = {}
+
+    def stub_function():
+        pass
+
+    with restrict_region.handle_custom_resource_status_message(event, context):
+        stub_function()
+
+    assert not mock_cfn_send.called
+
+
+def test_handle_custom_resource_status_message__failed_execution__not_custom_resource(mocker: MockerFixture):
+    mock_cfn_send = mocker.patch.object(cfnresponse, "send", autospec=True)
+
+    event = {""}
+    context = {}
+
+    def stub_function():
+        raise Exception("failed")
+
+    with pytest.raises(Exception):
+        with restrict_region.handle_custom_resource_status_message(event, context):
+            stub_function()
+    assert not mock_cfn_send.called
+
+
+def test_handle_custom_resource_status_message__successful_execution__is_custom_resource(mocker: MockerFixture):
+    mock_cfn_send = mocker.patch.object(cfnresponse, "send", autospec=True)
+
+    event = {"RequestType": "Create"}
```

Comment:
hmm, no test for "update" and "delete" events?

Reply:
In this scope, it should not matter what request type it is as long as it's not `None`